internal/internal_workflow_testsuite.go

```diff
@@ -52,6 +52,7 @@ const (
 	defaultTestWorkflowTypeName = "default-test-workflow-type-name"
 	defaultTestDomainName       = "default-test-domain-name"
 	workflowTypeNotSpecified    = "workflow-type-not-specified"
+	testHeader                  = "test-header"
```

Comment:
I think customers should be able to inject any headers into the workflow under the test.

Reply:
Removed, made this configurable so user can use on tests..